### PR TITLE
Added more indian channels

### LINKED ISSUE #303: Please add more indian channels
Please check the link given
http://mhdtvworld.com/

### PR DESCRIPTION
**Pull request by freearhey**
Resolve #303

### CODE REVIEW

File: channels/unsorted.m3u [hunk 1]
```diff
@@ -1 +1,379 @@
...
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/unnamed.jpg" group-title="Entertainment",Vasantham
+https://d39v9xz8f7n8tk.cloudfront.net//hls//vsnthmctv//master.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/b4u_plus_ae.png" group-title="Movies",B4U Plus
+http://livecdnh3.tvanywhere.ae/hls/b4uplus/04.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/zee-aflam-in.png" group-title="Movies",Zee Aflam
+http://livecdnh3.tvanywhere.ae/hls/zeeaflam/01.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/Screenshot_1.png" group-title="Movies",Sony Max HD
+https://msleurope2s.akamaized.net/hls/live/2000183/setmaxukhdmsl/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/Sony_Pix_HD.png" group-title="",Sony Pix HD
+http://xtreamsports.xyz/jio/jiotvxs.php?c=Sony_Pix_HD&q=600&e=.m3u8
```
Comment (adamo331) [inline channels/unsorted.m3u]: http://xtreamsports.xyz/jio/jiotvxxxxXX.php?c=Sony_Pix_HD&q=600&e=.m3u8

**Review by adamo331 — COMMENTED** [review]
http://xtreamsports.xyz/jio/jiotvxxxxXX.php?c=Sony_Pix_HD&q=600&e=.m3u8

File: channels/in.m3u [hunk 2]
```diff
@@ -323,413 +657,31 @@ http://45.79.203.234:1935/vendharm/myStream/playlist.m3u8
...
-#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/CTAMIL.png" group-title="Local",C Tamil TV
-http://ctamiltv.livebox.co.in/ctamiltvhls/live.m3u8
-#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/02/yuppclassicsmarathi.png" group-title="Movies",Classics Marathi
-https://d1fi19tywmn14b.cloudfront.net/yuppindmov/ngrp:ymarcla.stream_all/playlist.m3u8
-#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/unnamed.jpg" group-title="Entertainment",Vasantham
-https://d39v9xz8f7n8tk.cloudfront.net//hls//vsnthmctv//master.m3u8
-#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/b4u_plus_ae.png" group-title="Movies",B4U Plus
-http://livecdnh3.tvanywhere.ae/hls/b4uplus/04.m3u8
-#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/zee-aflam-in.png" group-title="Movies",Zee Aflam
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/zee-aflam-in.png" group-title="Movies",Zee Aflam
```
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/in.m3u]: "Zee Aflam is an Indian pay television channel..." (https://en.wikipedia.org/wiki/Zee_Aflam)

File: channels/in.m3u [hunk 3]
```diff
@@ -167,30 +319,106 @@ http://159.203.9.134/hls/jhanjar_music/jhanjar_music.m3u8
...
+http://stream.cinesoft.live/cinehome/cmovies_576p/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/11/MOVIECHANNEL.png" group-title="Movies",Movie Channel (Opt-2)
+http://185.59.223.60/MBO2/index.m3u8?token=RedflixTV
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/movies_now.png" group-title="Movies",Movies Now
+https://timesnow.airtel.tv/live/MN_pull/master.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/02/osn_music_now.png" group-title="Music",Music Now
+http://livecdnh1.tvanywhere.ae/hls/music_now/04.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="https://is4-ssl.mzstatic.com/image/thumb/Purple128/v4/83/74/55/8374557a-7e1b-dc56-e79a-7494fd40a275/source/60x60bb.jpg" group-title="",Namdhari
 https://namdhari.tv/live/sbs1.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/01/natgeo12_wild_us.png" group-title="",Nat Geo Wild HD
```
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/in.m3u]: Why do you think so?

File: channels/in.m3u [hunk 4]
```diff
@@ -333,3 +333,403 @@ http://103.199.161.254/Content/wowmusic/Live/Channel(WowMusic)/Stream(02)/index.
...
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/gauNDATV452.jpg" group-title="",Gaunda Punjab TV
+http://start.agmediachandigarh.com/gaundapunjab/tv/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/BHAGY5678.jpg" group-title="",Bhaagya TV
+http://live.wmncdn.net/bhaagya1/54f1d025e282fee369a8f2d44998dc49.sdp/mono.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/TENHD1.png" group-title="Sport",Sony Ten 1 HD (Opt-2)
+http://tv11.roarzone.info:8080/live/ten1-ebx/chunks.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/TENHD2.png" group-title="Sport",Sony Ten 2 HD (Opt-2)
+http://tv11.roarzone.info:8080/live/ten2-ebx/chunks.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/TENHD3.png" group-title="Sport",Sony Ten 3 HD (Opt-2)
+http://tv11.roarzone.info:8080/live/ten3-ebx/chunks.m3u8
```
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/in.m3u]: working now

File: channels/in.m3u [hunk 5]
```diff
@@ -333,3 +333,403 @@ http://103.199.161.254/Content/wowmusic/Live/Channel(WowMusic)/Stream(02)/index.
...
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/PUNJAB5AAB.jpg" group-title="",5aab TV
+http://158.69.124.9:1935/5aabtv/5aabtv/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/gauNDATV452.jpg" group-title="",Gaunda Punjab TV
+http://start.agmediachandigarh.com/gaundapunjab/tv/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/BHAGY5678.jpg" group-title="",Bhaagya TV
+http://live.wmncdn.net/bhaagya1/54f1d025e282fee369a8f2d44998dc49.sdp/mono.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/TENHD1.png" group-title="Sport",Sony Ten 1 HD (Opt-2)
+http://tv11.roarzone.info:8080/live/ten1-ebx/chunks.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/TENHD2.png" group-title="Sport",Sony Ten 2 HD (Opt-2)
+http://tv11.roarzone.info:8080/live/ten2-ebx/chunks.m3u8
```
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/in.m3u]: both working now

File: channels/in.m3u [hunk 6]
```diff
@@ -333,3 +333,403 @@ http://103.199.161.254/Content/wowmusic/Live/Channel(WowMusic)/Stream(02)/index.
...
+http://live.wmncdn.net/pbnmusic/b437d1e185b4c95b08b9c830a9d97cff.sdp/mono.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/DESICHANNEL09.jpg" group-title="Music",Desi Channel
+http://live.wmncdn.net/desichannel/7e2dd0aed46b70a5c77f4affdb702e4b.sdp/mono.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/PUNJAB5AAB.jpg" group-title="",5aab TV
+http://158.69.124.9:1935/5aabtv/5aabtv/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/gauNDATV452.jpg" group-title="",Gaunda Punjab TV
+http://start.agmediachandigarh.com/gaundapunjab/tv/playlist.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/BHAGY5678.jpg" group-title="",Bhaagya TV
+http://live.wmncdn.net/bhaagya1/54f1d025e282fee369a8f2d44998dc49.sdp/mono.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/12/TENHD1.png" group-title="Sport",Sony Ten 1 HD (Opt-2)
```
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/in.m3u]: Bhaagya TV channel removed

File: channels/unsorted.m3u [hunk 7]
```diff
@@ -682,4 +682,76 @@ http://rtmp.santhoratv.zecast.net/tamilbusinesstv/tamilbusinesstv/playlist.m3u8
...
\ No newline at end of file
+https://vidcdn.vidgyor.com/thanthi-origin/liveabr/thanthi-origin/live1/chunks.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/11/sun_chutti_tv_in.png" group-title="",Chutti TV
+http://xtreamsports.xyz/jio/jiotvxxxXXxXXxs.php?c=Chutti_TV&q=600&e=.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/11/sun_adithya.png" group-title="",Adithya TV
+http://xtreamsports.xyz/jio/jiotvxxxXXxXXxs.php?c=Adithya_TV&q=600&e=.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2018/11/star-vijay-international-hk.png" group-title="Entertainment",Star Vijay
+http://216.144.250.174/Star_VIjay_HD/index.m3u8
+#EXTINF:-1 tvg-logo="http://mhdtvworld.com/wp-content/uploads/2019/03/colorstamilhd.png" group-title="",Colors Tamil HD
+http://xtreamsports.xyz/jio/jiotvxxxXXxXXxs.php?c=Colors_Tamil_HD&q=600&e=.m3u8
```
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/unsorted.m3u]: removed

File: channels/unsorted.m3u (hunk 1 above)
Comment (freearhey) [reply channels/unsorted.m3u]: removed

File: channels/in.m3u (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamo331) [reply channels/in.m3u]: the source of m3u8 link for this channel is for myHD package UAE

File: channels/unsorted.m3u (hunk 1 above)
Comment (adamo331) [reply channels/unsorted.m3u]: look also to new file added there many india link from http://mhdtvsports.com are no working. i find many of them

Comment (freearhey) [reply channels/unsorted.m3u]: there are no such links in the current version of playlists

File: channels/in.m3u (hunk 3 above)
[the remark this answers is not in the document]
Comment (freearhey) [reply channels/in.m3u]: thanks for noticing! moved the channel to international playlist

**Pull request merged**